azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -109,6 +109,21 @@ public static boolean isStatusSucceeded(final Status status) {
     }
   }
 
+  public static boolean isStatusToLog(final Status status) {
```

Comment:
Perhaps a more generic name would be better suited, something like `isStatusFinishedWithoutSuccess`.

Reply:
Wonderful suggestion. Will fix.